src/raftstore/store/worker/region.rs

```diff
@@ -250,22 +362,80 @@ impl SnapContext {
         timer.observe_duration();
     }
 
-    fn handle_destroy(&mut self, region_id: u64, start_key: Vec<u8>, end_key: Vec<u8>) {
-        info!(
-            "[region {}] deleting data in [{}, {})",
-            region_id,
-            escape(&start_key),
-            escape(&end_key)
-        );
+    fn handle_destroy(
```

Comment:
s/handle_destroy/cleanup_range

Reply:
I think we should keep it, `handle destroy` is the original name, it matches task `destroy`.

Reply:
No, it has changed, you use `handle_destroy` multiple places use the semantics of cleanup a range, not destroy a peer.